docs/en/ingest-management/getting-started.asciidoc

```diff
@@ -1,159 +1,191 @@
+//REVIEWERS: I've added the Kibana requirements because they weren't stated. Let
+//me know if you'd rather that I leave those out. I personally have concerns
+//about telling users to disable TLS checks.
+
+* A user with the superuser role. See {ref}/built-in-roles.html[Built-in roles].
 
 [float]
 [[enable-ingest-management]]
 == Step 1: Enable {ingest-management}
 
-To enable {ingest-management}, make the following changes to the {kib} configuration:
-
-- set `xpack.ingestManagement.enabled` to `true`
+To enable {ingest-management}:
 
-If you are using our https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}] on
-{ecloud}, you will need to add them to the {kib} user settings as described in 
+. Add the following setting to the {kib} configuration:
++
+[source,yaml]
+----
+xpack.ingestManager.enabled: true
+----
++
+* On our
+https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}] on
+{ecloud}, add the setting to the {kib} user settings as described in 
 {cloud}/ec-manage-kibana-settings.html[Add {kib} user settings].
-
-If you are using an **on-premises** stack deployment, you will need to add them to the {kib} configuration
- as described in {kibana-ref}/ingest-manager-settings-kb.html[{ingest-manager} settings].
-
-To verify that {ingest-management} has been correctly enabled, login to {kib}, and select 
-**Management > {ingest-manager}**. If the menu entry is visible, and you see the following start page
-for {ingest-manager}, enabling {ingest-management} has been successful.
-
+* On a self-managed stack, add the setting to the {kib} configuration as
+described in
+{kibana-ref}/ingest-manager-settings-kb.html[{ingest-manager} settings].
+
+. Verify that {ingest-management} has been correctly enabled:
+.. Log in to {kib} and go to **Management > {ingest-manager}**.
++
+If the menu entry is visible, and you see the following start page
+for {ingest-manager}, {ingest-management} is successfully enabled.
++
 [role="screenshot"]
 image::images/kibana-ingest-manager-start.png[{ingest-manager} app in {kib}]
 
+//TODO: Add tabbed panel when the code is stable.
+
 [float]
 [[install-integration]]
 == Step 2: Install an integration and create a data source
 
-{ingest-management} bundles various assets that are needed to ingest and visualize data in **integrations**.
-
-In this Guide we assume that you have `nginx` running on some of your infrastructure, and want to
-collect logs and metrics from it. To do so, first install the `nginx` integration provided by Elastic.
+{ingest-manager} provides _integrations_ that bundle various assets needed to
+ingest and visualize data.
 
-In the {ingest-manager} app, navigate to **Integrations** and use the search bar to find the Nginx integration.
+In this guide, we assume that you have `nginx` running on some of your
+infrastructure, and want to collect logs and metrics from it. To do so:
 
+. Install the `nginx` integration provided by Elastic:
```

Comment:
When I wrote this, I had an extra step about installing the integration (can be done from the 'settings' section in the integration details page) and only then explained how to create the data source.

As creating the data source installs the integration in the background this step wasn't necessary and I removed it. Maybe this line should be rephrased or removed so that users don't look for an explicit installation step they don't need.